src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.spec.ts

```diff
@@ -239,7 +216,7 @@ describe('DotEditPageMainComponent', () => {
             dotContentletEditorService.close$.next(true);
             expect(dotRouterService.goToEditPage).toHaveBeenCalledWith({
                 url: '/about-us/index2',
-                language_id: mockDotRenderedPage().page.languageId.toString()
+                languageId: mockDotRenderedPage().page.languageId.toString()
```

Comment:
There was a typo here. Source: https://github.com/dotCMS/core-web/blob/master/src/app/api/services/dot-router/dot-router.service.ts#L74

Reply:
Are you sure about this? because in the endpoint the param we should pass is `language_id` for example in demo:

![image](https://user-images.githubusercontent.com/751424/104763927-4273e080-572c-11eb-9c0b-acc1e81569a4.png)

You tests the page in multiples languages with this change?